src/SpeedDial/SpeedDial.js

```diff
@@ -0,0 +1,144 @@
+// @flow weak
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { withStyles } from 'material-ui/styles';
+import Button from 'material-ui/Button';
+import Icon from 'material-ui/Icon';
+import AddIcon from 'material-ui-icons/Add';
```

Comment:
No, we don't rely on `material-ui-icons`. It needs to be an internal svg icons. We already have ~6 of them.

Reply:
Oops, I should know that! (SpeedDial started as a docs demo, and grew from there, so was using originally material-ui-icons)